### PR TITLE
Fix the organization list API of bitbucket

### PR DESCRIPTION
**Pull request by LinuxSuRen**
see the offical document https://developer.atlassian.com/cloud/bitbucket/rest/api-group-workspaces/#api-user-permissions-workspaces-get

### CODE REVIEW

File: scm/driver/bitbucket/testdata/workspaces.json.golden [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [
     {
         "Name": "atlassian",
-        "Avatar": "https://bitbucket.org/account/atlassian/avatar/32/"
+        "Avatar": "https://bitbucket.org/workspaces/atlassian/avatar"
     }
 ]
```
Comment (ankitm123) [inline scm/driver/bitbucket/testdata/workspaces.json.golden:6]: NIT: new line

**Comment by ankitm123** [comment]
Minor comment around missing new lines, but otherwise looks good!

File: scm/driver/bitbucket/testdata/workspaces.json.golden (hunk 1 above)
Comment (LinuxSuRen) [reply scm/driver/bitbucket/testdata/workspaces.json.golden]: Good catch!

**Pull request merged**